verifier/src/main/java/com/okta/jwt/JwtHelper.java

```diff
@@ -36,7 +36,8 @@
 public final class JwtHelper {
 
     private String issuerUrl;
-    private String clientOrAudience;
+    private String audience = "api://default";
```

Comment:
Why are we defaulting to this value? Aren't we throwing if there is no issuer or audience?